app/server/utils/transaction_limits.py

```diff
@@ -0,0 +1,140 @@
+from server.models import token
+
+
+def check_user_liquid_token_type(user, credit_transfer):
+    t = credit_transfer.token
+    if t is not None and t.token_type == token.TokenType.LIQUID:
+        return True
+
+    return False
+
+
+def check_group_user_liquid_token_type(user, credit_transfer):
+    t = credit_transfer.token
+    if t is not None and t.token_type == token.TokenType.LIQUID:
+        # TODO: add group account check
```

Comment:
maybe throw a `NotImplementedError` here for now?

Reply:
good idea